mappings/net/minecraft/client/gui/widget/ListWidget.mapping

```diff
@@ -1,11 +1,17 @@
-CLASS cwk net/minecraft/client/gui/widget/AbstractListWidget
+CLASS cwk net/minecraft/client/gui/widget/ListWidget
+	FIELD minecraft client Lcus;
+	FIELD x0 height I
+	FIELD x1 width I
+	FIELD y0 top I
+	FIELD y1 bottom I
+	FIELD yo scroll D
```

Comment:
These should be official names, although this improves the names.
Not the case with `client` tho.

Reply:
Yep, all these names need to go, including the client mapping.

Reply:
I would disagree pretty strongly there. The names Mojang has given them aren't much better than if they had just obfuscated them.